layouts/partials/section-nav.html

```diff
@@ -2,7 +2,7 @@
     <div class="container mx-auto uppercase">
         <ul>
             {{ range .sections }}
-                <li><a href="#{{ .id }}">{{ .label }}</a></li>
+                <li><a data-track="nav-{{ .label | urlize }}" href="#{{ .id }}">{{ .label }}</a></li>
```

Comment:
```suggestion
                <li><a data-track="nav" href="#{{ .id }}">{{ .label }}</a></li>
```

... since the href contains the target, and that should be enough to identify what was clicked. 

We should also set down some conventions for naming these, since they vary a bit between pages.